### ISSUE TITLE
Issues when using against Elasticsearch 7 with no document types

### ISSUE THREAD

**Issue by mattca** [federicotdn/inelastic#3]
When using this tool against Elasticsearch 7 I had the following issue when running against an index without a document type:
```
Traceback (most recent call last):
  File "inelastic.py", line 278, in <module>
    main()
  File "inelastic.py", line 261, in main
    args.verbose)
  File "inelastic.py", line 177, in get_inverted_index
    raise MissingDocumentTypeException(doc_type)
__main__.MissingDocumentTypeException: _doc
```
I applied the following hack to get past this:
```
diff --git a/inelastic.py b/inelastic.py
index 5785fad..c9b30cb 100755
--- a/inelastic.py
+++ b/inelastic.py
@@ -173,10 +173,10 @@ def get_inverted_index(es, index, doc_type, field, id_field, query, verbose):
         raise MissingIndexException(index)

     mappings = es.indices.get_mapping(index=index)[index]['mappings']
-    if doc_type not in mappings:
-        raise MissingDocumentTypeException(doc_type)
+#    if doc_type not in mappings:
+#        raise MissingDocumentTypeException(doc_type)

-    doc_mapping = mappings[doc_type]['properties']
+    doc_mapping = mappings['properties']

     if field not in doc_mapping:
         raise MissingFieldException(field)
```
Requires a bit more thought for a pull request though.

**Comment by federicotdn** [comment]
Hi @mattca, thanks for reporting this, will take a look.

**Comment by federicotdn** [comment]
Fixed in library version `0.2.2`! Use the `0.1.x` versions for Elasticsearch 6 and the `0.2.x` versions for Elasticsearch 7.
